src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -46,7 +46,7 @@ public static class DnnCatalog
         /// <remarks>
         /// The support for retraining is under preview.
         /// </remarks>
-        public static DnnRetrainEstimator RetrainDnnModel(
+        internal static DnnRetrainEstimator RetrainDnnModel(
```

Comment:
Should the `DnnRetrainTransformer` and `DnnRetrainEstimator` classes also be marked as `internal`?